src/server/render/default-document.js

```diff
@@ -16,7 +15,7 @@ const escapeScriptTags = (data) => {
     .replace(/\u2029/g, "\\u2029")
 }
 
-const defaultHtml = ({ markup, head, asyncProps, assets = {} }) => {
+const DefaultDocument = ({ html, css, head, props, assets }) => {
```

Comment:
I think using the name `props` when it's not a react component name's props but it _is_ a React component is a recipe for confusion. I'm sure we had a conversation with Matt once where we came up with a less react-treading-on-toes-y name.